go/api/commands.go

```diff
@@ -722,7 +722,7 @@ type ConnectionManagementCommands interface {
 	//  result, err := client.Ping("Hello")
 	//
 	// [valkey.io]: https://valkey.io/commands/ping/
-	Ping() (string, error)
```

Comment:
I propose to duplicate `ping` API (actually, entire `ConnectionManagementCommands`), because their docs and behaviour differ for standalone and cluster clients. Implementation should be duplicated too, unfortunately.

Reply:
See #2844